src/sentry/grouping/enhancer/enhancement-configs/newstyle@2023-01-11.txt

```diff
@@ -119,25 +119,53 @@ family:native package:libobjc* category=system
 family:native package:libsystem* category=system
 family:native package:/system/** category=system
 family:native package:/vendor/** category=system
-module:dalvik.system.* category=system
-module:com.android.* category=system
 family:native package:libdispatch.dylib category=system
 family:native package:WebKit category=system
 family:native package:**/libart.so category=system
 package:/apex/com.android.*/lib*/** category=system
 
+
+## java
+# Having a category prevents deriving in_app stack trace rules
+# XXX: Adjust all -app lines in the next PR
+module:akka.* category=system
+module:com.android.* category=system
+module:com.fasterxml.* category=system
+module:com.google.* category=system
+module:com.microsoft.* category=system
+module:com.sun.* category=system
+module:feign.* category=system
+module:dalvik.system.* category=system
+module:io.opentelemetry.* category=system
+module:io.sentry.* -app -group
```

Comment:
Some of the lines here come from the bottom of this file:
<img width="250" alt="image" src="https://github.com/user-attachments/assets/dba30e1f-78b8-4ab3-b43a-e7450b68a4e9" />